docs/tutorials/index.md

```diff
@@ -174,7 +174,7 @@ The Gluon and Module tutorials are in Python, but you can also find a variety of
 
 <div class="applications">
 
-- [Connectionist Temporal Classification](http://mxnet.incubator.apache.org/tutorials/speech_recognition/ctc.html)
+- [Connectionist Temporal Classification](../tutorials/speech_recognition/ctc.html)
```

Comment:
Maybe we should use relative links to avoid version issues and keep it as a general guideline for all other links?

Reply:
yes - this is a good practice. I will scan others and carefully do this in a different PR.